src/main/java/crawlercommons/sitemaps/sax/extension/VideoHandler.java

```diff
@@ -101,7 +101,36 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
         if (currAttr == null) {
             currAttr = new VideoAttributes();
         }
-        if (value.isEmpty()) {
+        if ("price".equals(localName)) {
```

Comment:
Looks like all this code is just moving from another location, so I'm not actually reviewing it.